tools/installed_agent_e2e.py

```
@@ -2265,6 +2522,256 @@ def run_cancellation(pentect: str) -> None:
+                "text",
+                "--no-session-persistence",
+                "--dangerously-skip-permissions",
+                "--tools",
+                "",
+                "--model",
+                "claude-sonnet-4-5",
+                "--settings",
+                str(input_settings),
+                "Reply with DONE.",
+            ]
+            process = subprocess.Popen(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)